README.md

```diff
@@ -0,0 +1,26 @@
+#Crassus page :cactus:
+
+![alt-text](Captura de pantalla de 2020-10-25 22-06-42.png)
+
+This is the upgraded version of Crassus page, create with vanilla Javascript, HTML and CSS.
```

Comment:
You can add the url to your first crassus, so the person who reads this can easily go to review it. 
```suggestion
This is the upgraded version of the first [Crassus page](https://github.com/susird/crassus-basic), created with vanilla Javascript, HTML and CSS.
```